app.js

```diff
@@ -0,0 +1,21 @@
+function validateEmail(email) {
+	const re = /\S+@\S+\.\S+/;
+	return re.test(email);
+}
+
+function checkEmail() {
+	const $errorMessage = $("#error");
```

Comment:
```suggestion
	const $errorMessage = $('#error');
```

Hajde da `"` koristimo samo za html atribute a u funkcijama `'`